File: Examples/runtimes/java/DynamoDbEncryption/src/test/java/software/amazon/cryptography/examples/TestScanErrorExample.java
package software.amazon.cryptography.examples;

import org.junitpioneer.jupiter.RetryingTest;

public class TestScanErrorExample {

  @RetryingTest(3)
  public void ScanError() {
    ScanErrorExample.ScanError(
      TestUtils.TEST_KMS_KEY_ID,
      TestUtils.TEST_DDB_TABLE_NAME
    );
  }
}

File: Examples/runtimes/python/DynamoDBEncryption/test/test_scan_error_example.py
# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0
"""Test scan error example."""
import pytest

from ..src.scan_error_example import scan_error
from .test_utils import TEST_DDB_TABLE_NAME, TEST_KMS_KEY_ID

pytestmark = [pytest.mark.examples]


def test_scan_error():
    """Test scan_error."""
    scan_error(TEST_KMS_KEY_ID, TEST_DDB_TABLE_NAME)
